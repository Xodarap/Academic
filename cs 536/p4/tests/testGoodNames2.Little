int x;
int y;

void f(double a, int c, int d) { return; }

int g(int x) { 
  int y; 
  return 0;
}

void foo() {
  int a;
  int b;
  int c;
  int B;
  
  foo();
  while (1) {}
  scanf("%d", &a);
  scanf("%d", &B);
  B = 1 < 3;
  a = a + b;
  a = a - b;
  a = a * b;
  a = a / b;
  c = ! c;
  c = c && 1;
  c = 0 || (!c);
  c = a == b;
  c = a != b;
  c = a < b;
  c = a > b;
  c = a <= b;
  c = a >= b;
  a = b;
  a = 50;
  a = g(3) + 1;
  c = 1;
  c = 0;
  a = - 50;
  
  if (a >= b) {
    int a;
    while (a > 0) {
      int a;
      a = b;
    }
  }
  a = a;
  if (b == -1) {
    x = 5;
    while (c) {y = y + 1;}
  }
  else {
    printf("test");
    a = 0;
  }
  a = 2147483647;		
  g(a);
  f(a,c,b);
} 

int b;

