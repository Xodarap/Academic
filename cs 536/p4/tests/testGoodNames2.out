int x;
int y;

void f(double a, int c, int d) {
  return;
}

int g(int x) {
  int y;
  return 0;
}

void foo() {
  int a;
  int b;
  int c;
  int B;
  foo(->void)();
  while (1) {
  }
  scanf("%d", &a(int));
  scanf("%d", &B(int));
  B(int) = (1<3);
  a(int) = (a(int)+b(int));
  a(int) = (a(int)-b(int));
  a(int) = (a(int)*b(int));
  a(int) = (a(int)/b(int));
  c(int) = (!c(int));
  c(int) = (c(int)&&1);
  c(int) = (0||(!c(int)));
  c(int) = (a(int)==b(int));
  c(int) = (a(int)!=b(int));
  c(int) = (a(int)<b(int));
  c(int) = (a(int)>b(int));
  c(int) = (a(int)<=b(int));
  c(int) = (a(int)>=b(int));
  a(int) = b(int);
  a(int) = 50;
  a(int) = (g(int->int)(3)+1);
  c(int) = 1;
  c(int) = 0;
  a(int) = (-50);
  if ((a(int)>=b(int))) {
    int a;
    while ((a(int)>0)) {
      int a;
      a(int) = b(int);
    }
  }
  a(int) = a(int);
  if ((b(int)==(-1))) {
    x(int) = 5;
    while (c(int)) {
      y(int) = (y(int)+1);
    }
  }
  else {
    printf("test");
    a(int) = 0;
  }
  a(int) = 2147483647;
  g(int->int)(a(int));
  f(double,int,int->void)(a(int), c(int), b(int));
}
int b;
